manifests/0000_08_cluster-dns-operator_03-cluster-operator.yaml

```diff
@@ -3,3 +3,7 @@ kind: ClusterOperator
 metadata:
   name: dns
 spec: {}
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
```

Comment:
Is it valid to have an entry for the operator in `.status.versions` when the available condition is not true? According to [the godoc](https://github.com/openshift/api/blob/f1796006a30cb15d8ffd7f0584bdf87bedaf9e4f/config/v1/types_cluster_operator.go#L59-L60), the version "must always match the Available condition."

Reply:
this manifest is not directly applied, it is only used to signal to the cluster version operator.